### PR TITLE
qa_crowbarsetup: Update handling of heat cfn image

### PR DESCRIPTION
**Pull request by toabctl**
The current approach with using crudini for modifying tempest.conf
is racy because the next chef-client run overrides the modifications.
So just upload the image and use the standard image name for the
heat image which is set within the tempest barclamp.

### CODE REVIEW

**Comment by toabctl** [comment]
NOTE: This PR needs https://github.com/crowbar/crowbar-openstack/pull/618

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -3443,6 +3432,26 @@ function oncontroller_run_integration_test()
     return $integrationret
 }
 
+function oncontroller_heat_image_setup()
+{
+    local image_url=http://$clouddata/images/SLES11-SP3-x86_64-cfntools.qcow2
+    # this is the standard name we use in the tempest barclamp. if you change the name
+    # yu may also want to set the new name in the barclamp
```
Comment (aplanas) [inline scripts/qa_crowbarsetup.sh]: 'you'

File: scripts/qa_crowbarsetup.sh [hunk 2]
```diff
@@ -3443,6 +3432,26 @@ function oncontroller_run_integration_test()
     return $integrationret
 }
 
+function oncontroller_heat_image_setup()
+{
+    local image_url=http://$clouddata/images/SLES11-SP3-x86_64-cfntools.qcow2
+    # this is the standard name we use in the tempest barclamp. if you change the name
```
Comment (aplanas) [inline scripts/qa_crowbarsetup.sh]: 'If'

Comment (toabctl) [inline scripts/qa_crowbarsetup.sh]: done

File: scripts/qa_crowbarsetup.sh (hunk 1 above)
Comment (toabctl) [inline scripts/qa_crowbarsetup.sh]: done

**Pull request merged**